Example/Example/src/AnalyticsManager.swift

```diff
@@ -29,11 +30,39 @@ class AnalyticsManager {
             self.clickstream = try Clickstream.initialise(networkConfiguration: networkConfigs,
                                                           constraints: constraints,
                                                           eventClassification: classification)
+            
+            self.setClickstreamTracker()
         } catch  {
             print(error.localizedDescription)
         }
     }
     
+    /// Set Clickstream Health Tracker
+    private func setClickstreamTracker() {
+        
+        let customerInfo = CSCustomerInfo(signedUpCountry: "India", email: "test@test.com", currentCountry: "91", identity: 105)
+        let sessionInfo = CSSessionInfo(sessionId: "1001")
+        let appInfo = CSAppInfo(version: "1.1.0")
+        let commonProperties = CSCommonProperties(customer: customerInfo, session: sessionInfo, app: appInfo)
+        let configs = ClickstreamHealthConfigurations(minimumTrackedVersion: "0.1", trackedVia: .internal)
+        
+        self.clickstream?.setTracker(configs: configs, commonProperties: commonProperties, dataSource: self, delegate: self)
+    }
+    
+    private func getAppInfo() -> CSAppInfo {
+        return CSAppInfo(version: "1.0")
+    }
+    
+    private func getCustomerInfo() -> CSCustomerInfo {
```

Comment:
nitpicking, instead of `Customer` it should be `User`

Reply:
No Mas, it is a DTO required by Health event.